tests/integration/api_v2/rpc/pendingTransactions.test.js

```diff
@@ -0,0 +1,204 @@
+			const response = await getTransactions({
+				includePending: true,
+				senderAddress: refTransaction.sender.address,
+			});
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			expect(response.result).toMap(resultEnvelopeSchema);
+			expect(result.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(result.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(result.meta).toMap(metaSchema);
+		});
+
+		it('known transaction senderUsername -> ok', async () => {
+			const response = await getTransactions({
+				includePending: true,
+				senderUsername: refTransaction.sender.username,
+			});
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			expect(response.result).toMap(resultEnvelopeSchema);
+			expect(result.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(result.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(result.meta).toMap(metaSchema);
+		});
+
+		it('known transaction senderPublicKey -> ok', async () => {
+			const response = await getTransactions({
+				includePending: true,
+				senderPublicKey: refTransaction.sender.publicKey,
+			});
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			expect(response.result).toMap(resultEnvelopeSchema);
+			expect(result.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(result.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(result.meta).toMap(metaSchema);
+		});
+
+		it('empty transaction id -> ok', async () => {
+			const response = await getTransactions({
+				includePending: true,
+				transactionId: '',
+			});
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			expect(response.result).toMap(resultEnvelopeSchema);
+			expect(result.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(result.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(result.meta).toMap(metaSchema);
+		});
+
+		it('empty transaction moduleAsset -> ok', async () => {
+			const response = await getTransactions({
+				includePending: true,
+				moduleAssetId: '',
```

Comment:
@MichalTuleja Same as above.

Reply:
@sameersubudhi Please look at the answer above.